service/resource/deployment/deployment.go

```diff
@@ -12,13 +10,8 @@ import (
 )
 
 const (
-	mainDeploymentName          = "cluster-main-template"
-	templatesBaseURI            = "https://raw.githubusercontent.com/giantswarm/azure-operator/%s/service/arm_templates/"
-	templateVersion             = "1.0.0.0"
-	mainTemplate                = "main.json"
-	clusterSetupTemplate        = "cluster_setup.json"
-	securityGroupsSetupTemplate = "security_groups_setup.json"
-	virtualNetworkSetupTemplate = "virtual_network_setup.json"
+	mainDeploymentName = "cluster-main-template"
+	templateVersion    = "1.0.0.0"
```

Comment:
Just for my interest, what is this for a version scheme?

Reply:
This comes from Azure.

Reply:
Ok, but what kind of format is this. Why 4 version parts?

Reply:
https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-authoring-templates search for `contentVersion`

Reply:
This link could be part of godoc.

Reply:
Done.